CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing Guidelines
+
+Thanks for your interest in contributing to `Thoth Station`.
+
+## Reporting Bugs
+
+Before creating bug reports, please check a [list of known issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Athoth-station+sort%3Acreated-asc) to see
+if the problem has already been reported (or fixed in a master branch).
+
+**Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+You can also comment on the closed issue to indicate that upstream should provide a new release with a fix.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as GitHub issues.
+When you are creating an enhancement issue, **use a clear and descriptive title** and **provide a clear description of the suggested enhancement** in as many details as possible.
+
+## Guidelines for Developers
+
+If you would like to contribute code to the `Thoth` project, this section is for you!
+
+### Is this your first contribution?
+
+Please take a few minutes to read GitHub's guide on [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/).
+It's a quick read, and it's a great way to introduce yourself to how things work behind the scenes in open-source projects.
+
+### Dependencies
+
+If you are introducing a new dependency, please make sure it's added to:
+
+- `Pipfile`
+- `setup.py` if its a package
+
+### Documentation
+
+If you want to update documentation, [README.md](README.md) is the file you're looking for.
+
+### How to contribute code to Project Thoth
+
+1. Create a fork of a repository.
+2. Create a new branch just for the bug/feature you are working on.
+3. Once you have completed your work, create a Pull Request, ensuring that it meets the requirements listed below.
+
+### Requirements for Pull Requests (PR)
+
+- Use `pre-commit` (see [below](#checkerslintersformatters--pre-commit)).
+- Use common sense when creating commits, not too big, not too small. You can also squash them at the end of review. See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).
+- Cover new code with a test case (new or existing one).
+- All tests have to pass.
+- Rebase against updated `master` branch before creating a PR to have linear git history.
+- Create a PR against the `master` branch.
```

Comment:
Can we add a line here about running mypy in an appropriate venv (if applicable)?

Reply:
isnt that mentioned a few lines below?

Reply:
ping

Reply:
Requiring external developers to have a proper dev setup when they want to contribute might be tricky or deterrent (given various versions of tools/linters available, their varying output across versions and different configurations available). Maybe just mentioning that each PR is checked on the CI side that has proper configuration might be better for new contributors, so they check results reported to the PR.